dist/mapdc.js

```diff
@@ -48641,6 +48646,31 @@ function rasterChart(parent, useMap, chartGroup, _mapboxgl) {
     return _legend;
   };
 
+  _chart.deleteLayerLegend = function (currentLayer, deleteLayer, prevLayerId) {
+    var legend = document.getElementsByClassName('legend');
```

Comment:
Should we be worried that starting the DOW tree search for `.legend` from the `document` could grab any other `.legend` element in the DOM outside of this chart? I think it should be scoped, something like `_chart.select(".legend")`

Reply:
@biovisualize, thanks for considering that. It probably does that. I will fix it.